python/src/QuantileConfidence_doc.i.in

```diff
@@ -0,0 +1,423 @@
+"Evaluate the minimum size of the sample for the unilateral case.
+
+When the rank :math:`r` is zero the minimum sample size is immediate for the lower bound:
+
+.. math::
+
+    n_{low} = \frac{\log(1-\beta)}{\log(1-\alpha)}
+
+and for the upper bound:
+
+.. math::
+
+    n_{up} = \frac{\log(1-\beta)}{\log(\alpha)}
+
+Else in the general case we search for the smallest :math:`n` which verifies:
+
+.. math::
+
+    F_{n,\alpha}(n-r-1) \geq \beta
+
+with :math:`F_{n,\alpha}` the cumulative distribution function of the Binomial distribution.
+
+Parameters
+----------
+rank : int, optional
+    Rank of the quantile. Default value is 0.
+tail : bool, optional
+    Whether to return the lower or upper tail minimum size. Default is False.
+
+Returns
+-------
+size : int
+    Minimum sample size
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> size = algo.computeUnilateralMinimumSampleSize(0)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeBilateralMinimumSampleSize
+"Evaluate the minimum size of the sample for the bilateral case.
+
+The minimal bilateral sample size is the smallest integer :math:`n` such that:
+
+.. math::
+
+    1-\alpha^n-(1-\alpha)^n \geq \beta
+
+if and only if
+
+.. math::
+
+    \Pset(X_(1) \leq x_{\alpha} \leq X_({n})) \geq \beta
+
+the size :math:`n` is searched inside the bounds given by the relationship:
+
+.. math::
+
+    \left\lfloor \frac{\log(1 - \beta)}{\log(\gamma)} \right\rfloor \leq n \leq \left\lceil \frac{\log(\frac{1-\beta}{2})}{\log(\gamma)} \right\rceil
+
+where
+
+.. math::
+
+    \gamma = max(\alpha, 1- \alpha)
+
+Returns
+-------
+size : int
+    Minimum sample size :math:`n`
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> size = algo.computeBilateralMinimumSampleSize()
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeAsymptoticBilateralRank
+"Evaluate the asymptotic bilateral rank of a quantile.
+
+The aymptotic bilateral ranks :math:`k_1, k_2 \in \llbracket 0, n - 1 \rrbracket` is estimated from:
+
+.. math::
+
+    k_1 & = \left\lfloor n \alpha - \sqrt{n} z_{1-\beta/2} \sqrt{\alpha (1 - \alpha)} \right\rfloor - 1 \\
+    k_2 & = \left\lfloor n \alpha + \sqrt{n} z_{1-\beta/2} \sqrt{\alpha (1 - \alpha)} \right\rfloor - 1
+
+with :math:`z_{1-\beta/2}` the standard Gaussian quantile of order :math:`1-\beta/2`, see [delmas2006]_ proposition 12.2.13 page 257.
+
+Parameters
+----------
+size : int
+    Sample size
+
+Returns
+-------
+ranks : :class:`~openturns.Indices` of size 2
+    Pair of lower and upper ranks :math:`k1, k2 \in \{0, \hdots, n-1\}^2`
```

Comment:
The formatting is not correct. Moreover, the fact that $k_1 \leq k_2$ should be emphasized.

```suggestion
    Pair of lower and upper ranks :math:`(k_1, k_2)` with :math:`0 \leq k_1 \leq k_2 \leq n-1`.
```